nxc/modules/hyperv-host.py

```diff
@@ -0,0 +1,53 @@
+from impacket.dcerpc.v5.rpcrt import DCERPCException
+from impacket.dcerpc.v5 import rrp
+from impacket.examples.secretsdump import RemoteOperations
+
+
+class NXCModule:
```

Comment:
You could add yourself as the module creator if you would like :) Take a look at https://github.com/Pennyw0rth/NetExec/blob/c0adbf1eb8cf7c2506955c642306fcde8782b851/nxc/modules/winscp.py#L19 as an example